### ISSUE TITLE
Cadmus all of a sudden stopped working, it has no sound output.

### ISSUE THREAD

**Issue by IAmV0id** [josh-richardson/cadmus#43]
It was working earlier today, I went to sleep and when I woke up. I couldn't get it to work, It didn't output any sound. I'm not sure what happened because I never had any problems with it before, I've spent a few hours messing around trying to get it to work. I'm using Pop!_OS 20.04

**Comment by sebastiandg7** [comment]
Same here! Pop OS 20.04. No output from `Cadmus Denoised Output` device

**Comment by bold-sama** [comment]
me and a buddy are also dealing with this on manjaro kde 20.0.3

**Comment by bigbruno** [comment]
I'm found project yesterday and not working in Kubuntu 20.04 updated.

Show device Cadmus Denoised Output, but always without any sound.

**Comment by Frozen-byte** [comment]
same for me: audio sink is created but is totally quiet

Distro: kubuntu 19.10

**Comment by Frozen-byte** [comment]
maybe related to: https://github.com/josh-richardson/cadmus/issues/32

**Comment by homamgamer** [comment]
the Fix might be #40

**Comment by livkx** [comment]
Built .deb available for anyone who needs it at https://github.com/livkx/cadmus/releases/tag/0.0.3

This repo seems abandoned so I'll maintain it myself on my fork until the author returns.

**Comment by rbertoche** [comment]
Since no one had the same problem I had I'm posting here:
I had no sound even running version 0.0.3. To make things worse, in KDE/Plasma the menu doesn't have a slider - IMHO thats the way the menu appliance is expected to work - which put me looking in the wrong direction.
Turns out I needed to unmute a loopback entry in the Recording tab of pavucontrol. After that it started working.

**Comment by livkx** [comment]
Yeah, the slider doesn't show on KDE, I'm still working out why that's happening. Not sure why you needed to unmute something in pavu unless you'd previously muted it yourself?

**Comment by deeoxys** [comment]
Any update on this?
Complete silence on pop!_os 21.10.
